.github/workflows/test.yml

```diff
@@ -22,5 +23,3 @@ jobs:
           pnpm clean
           pnpm build
           pnpm test-packaging
-        env:
```

Comment:
CI is always true

https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables